packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/encryption.js

```diff
@@ -144,15 +144,21 @@ describe('plugin-conversation', () => {
       });
 
       describe('#cardAction()', () => {
-        it('creates a cardAction Activity', () => spark.internal.conversation.cardAction(conversation, {
-          objectType: 'submit',
-          inputs: {key: 'value'}
-        })
+        it('creates a cardAction Activity', () => spark.internal.conversation.post(conversation, {displayName: 'First Message', cards: ['test']})
           .then(() => checkov.spark.internal.conversation.get(conversation, {activitiesLimit: 1}))
           .then((c) => {
             assert.property(c, 'defaultActivityEncryptionKeyUrl');
-            /* eslint-disable no-unused-expressions */
-            expect(c.activities.items[0].object.inputs).to.not.be.null;
+            assert.equal(c.activities.items[0].object.displayName, 'First Message');
+            spark.internal.conversation.cardAction(conversation, {
+              objectType: 'submit',
+              inputs: {key: 'value'}
+            }, c.activities.items[0])
```

Comment:
This is fine. It might be more clear to comment or have this as a `parent` variable, but not a blocker.